clubs/README.md

```diff
@@ -50,66 +49,64 @@ _(each of the sections below corresponds with a bullet point)_
 
 ### Leadership Team
 
-You'll want 2-3 people on the primary leadership team running the club. When
-choosing your leadership team, we recommend working with close friends who you
-think have strong leadership skills. It's also a good idea to have a mix of
-genders on the leadership team. Correlation doesn't imply causation, but the
-club member demographics mirror the demographics of the leadership team in many
-clubs.
+You'll want 2-3 people on the leadership team running the club. When choosing
+your team, we recommend working with close friends who you think have strong
+leadership skills. It's a good idea to have a mix of genders on the team.
+Correlation doesn't imply causation, but the club member demographics mirror the
+demographics of the leadership team in many clubs.
 
-Also, leadership skills are _far_ more important than technical skills. A common
-pitfall many clubs run into is selecting the leaders based on their technical
-skills rather than their leadership capacity.
+When selecting the team, leadership skills are _far_ more important than
+technical abilities. A common pitfall many clubs run into is selecting the
+leaders based on their technical skills rather than their leadership capacity.
 
 **Great clubs have been started by leaders that have never coded before.**
 
-When putting together the leadership team, make sure to choose one leader who's
-"in charge" (this will generally be you). This will be the person that has the
-final authority to make changes in the club.
+When putting together the leadership team, choose one leader who's "in charge"
+(this will usually be you). This'll be the person that has the final authority
```

Comment:
Fixed, thanks!

On Sun, Jan 17, 2016 at 2:02 AM, Max Wofford <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/658#discussion_r49944014>:
>
> >
> >  **Great clubs have been started by leaders that have never coded before.**
> >
> > -When putting together the leadership team, make sure to choose one leader who's
> > -"in charge" (this will generally be you). This will be the person that has the
> > -final authority to make changes in the club.
> > +When putting together the leadership team, choose one leader who's "in charge"
> > +(this will usually be you). This'll be the person that has the final authority
>
> "This'll be the person that has the final authority" -> "This person will
> have the final authority"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/658/files#r49944014>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta